receiver/mysqlreceiver/metadata.yaml

```diff
@@ -75,6 +75,16 @@ attributes:
   index_name:
     value: index
     description: The name of the index.
+  digest:
+    value: digest
+    description: Digest.
+  digest_text:
+    value: digest_text
+    description: Text before digestion.
+  event_states:
+    value: event_states
+    description: Possible event states.
+    enum: [star, timer_wait, errors, warnings, rows_affected, rows_sent, rows_examined, created_tmp_disk_tables, created_tmp_tables, sort_merge_passes, sort_rows, no_index_used]
```

Comment:
`timer_wait` is in milliseconds, can I mark it somehow?

Reply:
Attributes should not have units of their own. Units describe the numeric value of the metric, so are associated with the metric itself. 

I suspect we should have a separate metric for this special data point.

Reply:
Thanks for clarification, extracted as separate metric